base/dict.jl

```diff
@@ -498,12 +498,12 @@ end
 Return the value stored for the given key, or if no mapping for the key is present, return
 `f()`.  Use [`get!`](@ref) to also store the default value in the dictionary.
 
-This is intended to be called using `do` block syntax
+This is intended to be called using `do` block syntax, when the first argument
```

Comment:
Not sure the extra thing about "longer than one-liner" needs to be added. The example below immediately contradicts it.

Reply:
> Not sure the extra thing about "longer than one-liner" needs to be added.

Maybe just put a reference to `do block syntax`?

Reply:
Or "more-than-trivial throwaway functions". Not sure if that adds clarity or appropriate in the documentation.